src/Framework/AspNetCoreAnalyzers/src/Analyzers/Resources.resx

```diff
@@ -327,4 +327,10 @@
   <data name="Analyzer_PublicPartialProgramClass_Title" xml:space="preserve">
     <value>Unnecessary public Program class declaration</value>
   </data>
+  <data name="Analyzer_KestrelShouldListenOnIPv6AnyInsteadOfIpAny_Title" xml:space="preserve">
+    <value>Consider using IPAddress.IPv6Any instead of IPAddress.Any</value>
```

Comment:
```suggestion
    <value>Consider using ListenAnyIP instead of IPAddress.Any</value>
```

Reply:
should it be `Consider using ListenAnyIP() instead of Listen(IPAddress.Any)`? It feels like a proposition to change method name replacing the variable

Reply:
Sounds good to me.